peps/pep-9999.rst

```diff
@@ -72,8 +70,20 @@ proposal.
 Backwards Compatibility
 =======================
 
-As aliases in :external+py3.14:mod:`math` will be kept for indefinite time
-(their use would be discouraged), there are no anticipated code breaks.
+As aliases in :external+py3.14:mod:`math` will be kept indefinitely, there are
+no anticipated code breaks.  (The use of these aliases is discouraged.)
+
+Rationale
+=========
+
+Why not simply update the documentation to broaden math's scope
+---------------------------------------------------------------
+
+Broadening ``math``'s scope would solve the documentation issue, but it would
+neglect the motivation of providing a shared context.  As mentioned, shared
+context reduces verbosity in the documentation and conceptual load.  It also
+aids discoverability through grouping related functions and IDEs suggesting
+helpful completions.
```

Comment:
For me it looks like you repeat stuff from the "Motivation" section.  I hope, that with a small improvement of the former we can just remove this section.

Reply:
Oh, I'm happy to remove the Rationale.  I thought you wanted it.  Should I remove it now?